packages/nameguard-react/src/utils/text.ts

```diff
@@ -23,7 +23,8 @@ export function checkResultCodeTextColor(
         : "text-yellow-500";
       return color;
```

Comment:
There's no need to repeat `return color` on each case. Just put that once after the whole switch statement.